src/styles/components/_hero.scss

```diff
@@ -7,12 +7,15 @@
         margin-left: var(--offset-1);
         height: 100%;
         position: relative;
+
+        @include media('<=tablet') {
+            margin-left: 0;
+        }
     }
 
     &__img {
         height: 100%;
         object-fit: cover;
-        filter: brightness(60%);
```

Comment:
Where's the alternative code? 🤔

Reply:
Nevermind. I forgot I removed the background on the prototype hahaha Sorry. My bad :)